### PR TITLE
Update for Body Cams

### PR DESCRIPTION
**Pull request by Surrealaser**
## Changelog
:cl:
add: Security robots now have cams that are added to the security bodycam network.
tweak: Prevented mindless mobs wearing security uniforms from being added to and spamming the bodycam network.
/:cl:

### CODE REVIEW

**Comment by Surrealaser** [comment]
@TheSwain @remove32 Should be ready for merge, but feel free to review.

**Comment by Abraxis12** [comment]
Love it! More cams means more eyes in the sky, so to speak.

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 1]
```diff
@@ -289,9 +292,7 @@ Auto Patrol: []"},
 
 	log_combat(src,C,"stunned")
 	if(declare_arrests)
-		var/area/location = get_area(src)
-		speak("[arrest_type ? "Detaining" : "Arresting"] level [threat] scumbag <b>[C]</b> in [location].", radio_channel)
-
+		speak("[arrest_type ? "Detaining" : "Arresting"] level [threat] scumbag <b>[C]</b> in <b>[get_area(src)]</b>.", radio_channel)
 		arrest_security_record(C, arrest_type, threat, location) //FULPSTATION IMPROVED RECORD SECURITY PR -Surrealistik Oct 2019; this makes a record of the arrest, including timestamp and location.
```
Comment (remove32) [inline code/modules/mob/living/simple_animal/bot/secbot.dm]: you deleted the location var that this line references so this is failing to compile

**Comment by Surrealaser** [comment]
@remove32 @TheSwain Alright, should be good to go now; forgot to account for one instance of a var removal.

**Comment by remove32** [comment]
@TheSwain this bug i found is the kind of thing travis CI can catch for us

**Comment by Shadowtail117** [comment]
Borgs can already be viewed from applicable camera consoles, right? This seems redundant.

**Comment by Surrealaser** [comment]
@Shadowtail117 Secbots aren't borgs.

**Comment by Shadowtail117** [comment]
I am apparently legally blind. Whoops.

**Review by gonenoculer5 — APPROVED** [review]
I see no issues with this so far but im also tired as hell so feel free to vibe check me if I missed something. 👍

**Comment by remove32** [comment]
Testmerge was successful. Lets merge this shit

**Review by remove32 — APPROVED** [review]

File: code/modules/mob/living/simple_animal/bot/secbot.dm (hunk 1 above)
Comment (Surrealaser) [reply code/modules/mob/living/simple_animal/bot/secbot.dm]: Odd, compiling fine on my end.

Comment (remove32) [reply code/modules/mob/living/simple_animal/bot/secbot.dm]: this is an outdated comment. its good as it is now

**Pull request merged**